v21.1/create-table.md

```diff
@@ -764,6 +668,205 @@ Note that the foreign key constraint `fk_owner_id_ref_users` in the source table
+The `REGIONAL BY TABLE` locality is useful for tables that require low-latency reads and writes from specific region.
+
+For example, suppose you want to create a table for your application's end users in a specific state:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE TABLE users_ny (
+    id UUID PRIMARY KEY,
+    name STRING,
+    address STRING)
+    LOCALITY REGIONAL BY TABLE IN "us-east1"
+  ;
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT * FROM [SHOW TABLES] WHERE table_name='users_ny';
+~~~
+
+~~~
+  schema_name | table_name | type  | owner | estimated_row_count |            locality
+--------------+------------+-------+-------+---------------------+----------------------------------
+  public      | users_ny   | table | demo  |                   0 | REGIONAL BY TABLE IN "us-east1"
+(1 row)
+~~~
+
+{{site.data.alerts.callout_success}}
+`LOCALITY REGIONAL` is an alias for `LOCALITY REGIONAL BY TABLE`.
+{{site.data.alerts.end}}
+
+#### Create a table with a regional-by-row locality
+
+To create a table with a [`REGIONAL-BY-ROW`](multiregion-overview.html#regional-by-row-tables) locality, add a `LOCALITY REGIONAL BY ROW` clause to the end of the `CREATE TABLE` statement.
+
+The `REGIONAL BY ROW` locality is useful for tables that require low-latency reads and writes from different regions, where the low-latency region is specified at the row level.
+
+For example, the `vehicles` table of the [`movr` database](movr.html) is read to and written from nodes in different regions.
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE TABLE vehicles (
+    id UUID PRIMARY KEY,
+    type STRING,
+    city STRING,
+    owner_id UUID,
+    creation_time TIMESTAMP,
+    status STRING,
+    current_location STRING,
+    ext JSONB)
+    LOCALITY REGIONAL BY ROW
+  ;
+~~~
+
+CockroachDB will automatically assign each row to a region, based on the locality of the node from which the row is inserted. It will then optimize subsequent read and write queries executed from nodes located in the region assigned to the rows being queried.
+
+{{site.data.alerts.callout_info}}
+If the node from which a row is inserted has a locality that does not correspond to a region in the database, then the row will be assigned to the database's primary region.
+{{site.data.alerts.end}}
+
+To assign rows to regions, CockroachDB creates and manages a hidden `crdb_region` column, of [`ENUM`](enum.html) type `crdb_internal_region`. To override the automatic region assignment and choose the region in which rows will be placed, you can provide a value for the `crdb_region` column in `INSERT` and `UPDATE` queries on the table.
+
+{{site.data.alerts.callout_info}}
+The region value for `crdb_region` must be one of the regions added to the database, and present in the `crdb_internal_region` `ENUM`. To return the available regions, use a [`SHOW REGIONS FROM DATABASE <database name>`](show-regions.html) statement, or a [`SHOW ENUMS`](show-enums.html) statement.
```

Comment:
Could add some cross-linkage to more docs on `crdb_region` in https://www.cockroachlabs.com/docs/v21.1/set-locality#crdb_region